Lesson3/Lesson3/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Diagnostics;
+
+internal class Program
+{
+    private static void Main(string[] args)
+    {
+        Console.WriteLine("Lesson3\n");
+
+        int X = 0, Y = 0;
+        Console.Write("Enter an integer X: ");
+        bool XisTrue = int.TryParse(Console.ReadLine(), out X);
+        Console.Write("Enter an integer Y: ");
+        bool YisTrue = int.TryParse(Console.ReadLine(), out Y);
+
+        if (XisTrue && YisTrue)
+        {
+            int result = 0;
+            for (int i = (X > Y ? Y : X); i != (X < Y ? Y : X) + 1;)
```

Comment:
Doesn't look obvious, but I suspect that it works :) Next time you can try to move the logic of calculating the starting point and the end point out of the cycle